charts/steward/data/pipelineruns-default-networkpolicy.yaml

```
@@ -21,7 +21,7 @@ spec:
         - 10.0.0.0/8 # address block for private networks
         - 100.64.0.0/10 # shared address space for carrier-grade NAT
         - 169.254.0.0/16 # address block for local links; used for metadata.google.internal
-        - 172.16.0.0/12 # address block for private networks; used for K8s masters
+        - 172.16.0.0/12 # address block for private networks; used for K8s instance
```

A K8s instance consists of different parts each using different IP address ranges. Here we mean the control plane:
```suggestion
        - 172.16.0.0/12 # address block for private networks; used for K8s control plane
```